dynawo/sources/Models/CPP/ModelNetwork/test/TestLine.cpp

```diff
@@ -1150,8 +1150,9 @@ TEST(ModelsModelNetwork, ModelNetworkDynamicLine) {
   yTypesSave = yTypes;
   ASSERT_NO_THROW(dl->updateYType());
   ASSERT_NO_THROW(dl->updateFType());
-  ASSERT_EQ(fTypes, fTypesSave);
-  ASSERT_EQ(yTypes, yTypesSave);
+  ASSERT_EQ(fTypes[0], fTypesSave[0]);
+  ASSERT_EQ(yTypes[0], yTypesSave[0]);
```

Comment:
What about the index #1?

Reply:
I can definetely add it. The idea was that the logic is the same for index 0 and 1 (they are set by the same if loop) and so I just test 0 and 2 only for the yType. 
Let me know if you prefer that I add it